modules/manage/pages/security/fips-compliance.adoc

```diff
@@ -0,0 +1,42 @@
+= Configure Redpanda for FIPS 
+:description: Configure Redpanda to operate in FIPS compliance mode.
+:page-aliases: security:fips-compliance.adoc
+:page-categories: Management, Security
+
+Redpanda platform (Redpanda Enterprise) provides FIPS-compliant cipher enforcement for brokers using OpenSSL 3.0.9, which is https://csrc.nist.gov/projects/cryptographic-module-validation-program/certificate/4282[validated^] for https://csrc.nist.gov/pubs/fips/140-2/upd2/final[140-2^] and is undergoing validation by NIST for https://csrc.nist.gov/pubs/fips/140-3/final[140-3^]. Both Redpanda and `rpk` leverage validated Open SSL libraries for all security-related cryptography operations.
+
+== Prerequisites
+
+Before configuring brokers to run in FIPS compliance mode (FIPS mode), check to make sure the `redpanda-rpk-fips` and `redpanda-fips` packages are xref:deploy:deployment-option/self-hosted/manual/production/dev-deployment.adoc#install-fips[installed]. These packages are required by both the `redpanda` and `redpanda-tuners` install packages.
+
+== Configure FIPS
+
+When you configure a broker to run in FIPS mode:
+
+* FIPS compliance is enforced _immediately_ across all services upon startup.
+
+* Redpanda and its dependencies only use FIPS-validated cryptographic modules for all cryptographic algorithms used in a security context.
+
+Redpanda logs an error and exits immediately if:
+
+* The underlying operating system and crypto module are not running in FIPS mode.
+
+* The underlying cryptography module enters into an error state.
+
+* It cannot detect a FIPS-validated library.
+
+Additionally, a Redpanda broker will not join a cluster of brokers running in a different FIPS mode of operation. Instead, it logs an error and exits immediately.
```

Comment:
This is not true.  Redpanda clusters can be a heterogeneous mix of fips and non-fips nodes.  That's the purpose of the "non_homogenous_fips_mode" metric.

Reply:
metric or property?

Reply:
gauge metric.... currently in review (https://github.com/redpanda-data/redpanda/pull/19821)